packages/chain-adapters/src/evm/EvmBaseAdapter.ts

```diff
@@ -179,6 +182,11 @@ export abstract class EvmBaseAdapter<T extends EvmChainId> implements IChainAdap
         symbol: 'MATIC',
         explorer: 'https://polygonscan.com/',
       },
+      [KnownChainIds.GnosisMainnet]: {
+        name: 'Gnosis',
+        symbol: 'XDAI',
```

Comment:
https://docs.gnosischain.com/about/tokens/xdai#properties
```suggestion
        symbol: 'xDAI',
```